src/index.js

```diff
@@ -26,7 +26,7 @@ async function loadTranslations() {
     try {
         await i18next.init({
             interpolation: { escapeValue: false },
-            lng: 'en',
+            lng: 'es',
```

Comment:
we are switching default language to es?

Reply:
It get's overwritten as soon as the header loads, setting English as default language unless there is a previous selection from the user in local storage (from this dapp or rlogin).

See line #175:
https://github.com/money-on-chain/stable-protocol-interface-v2/blob/roc_748_featStaking_visuals/src/components/Header/index.jsx